core/kazoo_auth/src/kz_auth_util.erl

```diff
@@ -0,0 +1,99 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2012-2016, 2600Hz, INC
+%%% @doc
+%%%
+%%% @end
+%%% @contributors
+%%%-------------------------------------------------------------------
+-module(kz_auth_util).
+
+-include("kazoo_auth.hrl").
+
+%% ====================================================================
+%% API functions
+%% ====================================================================
+-export([map_keys_to_atoms/1]).
+
+-export([get_json_from_url/1, get_json_from_url/2]).
+
+-export([fetch_access_code/2, fetch_access_code/3]).
+
+-spec map_keys_to_atoms(map()) -> map().
+map_keys_to_atoms(Map) ->
```

Comment:
I'd much rather we have something along jsx's attempt_atom https://github.com/talentdeficit/jsx#decode12